packages/compat/src/compat-app-builder.ts

```diff
@@ -1365,18 +1364,6 @@ export class CompatAppBuilder {
   }
 }
 
-function maybeReplace(dom: JSDOM, element: Element | undefined): Node | undefined {
-  if (element) {
-    return definitelyReplace(dom, element);
-  }
-}
-
-function definitelyReplace(dom: JSDOM, element: Element): Node {
-  let placeholder = dom.window.document.createTextNode('');
-  element.replaceWith(placeholder);
-  return placeholder;
-}
-
```

Comment:
As far as I can tell these are somewhat duplicated by the same thing in core (previously known as `NodeRange`).